### ISSUE TITLE
Stuck on loading screen with OnePlus 8 Pro

### ISSUE THREAD

**Issue by jpsiitonen** [gskinnerTeam/flutter-wonderous-app#6]
Downloaded app from Play Store (v. 1.7.0).  
Tried to launch the app but it stays on loading screen and not going forward to home screen.

After a few tries, just full grey screen ( assuming it is flutter error ) 

Device: One Plus 8 Pro with Android 12.

**Comment by jonasbark** [comment]
same on Galaxy Fold 4 with Android 12L

**Comment by jpsiitonen** [comment]
This is most likely related to device language/locale setting. In fact using device with Finnish language it doesn't work, but with language English it's working.

**Comment by szerner** [comment]
Same issue on my Samsung A52 @Android 12, german. So I uninstalled the app, switched system language to English, installed again and app starts without issues. Reverted back to german system language and it's stuxk again while showing the splash screen / icon.

**Comment by gskinner** [comment]
Yes, unfortunately there is a known bug in the app with non-English languages on Android. We caught it right shortly launch and submitted a fix, but it has been waiting for review on the Play store for about 30 hrs now. Hopefully it will show up very soon.

Apologies for the poor initial experience.

**Comment by gskinner** [comment]
Update: It looks like the new version (1.8.0) has finally made it through review. Please update, and let us know if you still experience any issues with startup. Thank you so much for your patience!

Note that it may take a little while for the update to become available in your region.

**Comment by jpsiitonen** [comment]
Seems to work fine with 1.8.0 version.

**Comment by szerner** [comment]
@gskinner Now works for me on my german system as well. Thanks for fixing and giving us insight into this 'wonderous' app!

**Comment by gskinner** [comment]
Thanks @jpsiitonen & @szerner for checking. I hope you enjoy the app! Closing this issue.
